server/util/claims/claims.go

```diff
@@ -336,7 +337,11 @@ type ClaimsCache struct {
 	lru interfaces.LRU[*Claims]
 }
 
-func NewClaimsCache(ctx context.Context, ttl time.Duration) (*ClaimsCache, error) {
+func NewClaimsCache(ctx context.Context) (*ClaimsCache, error) {
```

Comment:
nit: can you remove ctx from here (and on up the chain) too?

Reply:
Done